.github/actions/api/breaking-change/action.yaml

```diff
@@ -0,0 +1,24 @@
+name: 'OpenAPI Spec: check for breaking API changes'
+description: 'Detect breaking changes in OpenAPI Specification 3'
+inputs:
+  base:
+    description: 'Path of original OpenAPI spec in YAML or JSON format'
+    required: true
+  revision:
+    description: 'Path of revised OpenAPI spec in YAML or JSON format'
+    required: true
+  fail-on-diff:
+    description: 'Fail with exit code 1 if any breaking changes are found'
+    required: false
+    default: 'true'
```

Comment:
Does this need to be a boolean-as-string value?  If so consider documenting it; if not then I prefer `true` (and then things get really weird because YAML has strange ideas about implicit conversions to boolean).